wcag-ruleset-review-process.md

```diff
@@ -1,77 +1,89 @@
 
 Much of the tasks in this process are delegated to ACT Task Force, which is a sub-group of AGWG. ACT TF carries out initial review and prepares proposals for AGWG, which has the final say as the maintainer of WCAG and the authoritative body for interpretation.
 
-Organizations and individuals, **rule providers**, can submit rules to be added to the WCAG Ruleset. Submitted rules must conform to the ACT Rules Format 1.0 specification and be accompanied by real-life implementations (specific criteria detailed later in this document). **Rule providers are expected to only contribute complete rules, as accurate as the rule providers can possibly make them.**
+The ACT TF does this work in close collaboration with the ACT Rules Community Group through regular joint meetings. Any other groups interested to contribute rules to the WCAG ruleset should contact the co-facilitator(s) of the ACT Task Force. The ACT publication process is designed to publish high-quality ACT rules to the W3C website. 
 
-## Rule Review
+## ACT Publication steps
 
-When proposing a new rule, or an update to a rule, there are three steps to go through:
+When creating a new rule, or updating a rule, there are five steps to go through. This process involves the combined effort of the ACT Task Force, and the ACT Rules Community Group. 
 
-1. **Submitting**: The rule provider submits a rule to ACT Task Force
-2. **Surveying**: The ACT Task Force reviews the rule through a survey
-3. **Approving**: The ACT Task Force prepares a proposal for AGWG
+1. Draft proposal
+2. Call for review
+3. Call for implementation
+4. Validation
+5. Publish approval
 
-The ACT Task Force will not edit rule contents. This is to avoid creating forks, where two versions of a rule are worked on in parallel, potentially with conflicts. Where necessary, the ACT Task Force will propose changes to the rule but not put those changes in.
+### 1. Draft proposal
 
-### Submitting
+New rules, and changes to rules are made through GitHub pull requests. Any member of either ACT group can author such a pull request. A pull request must have approval from three independent reviewers before it can proceed to step 2. These reviewers can not work for the same organization, and must not have been involved in writing the pull request. 
 
-The rule provider creates an issue in the [ACT GitHub repository](https://github.com/w3c/wcag-act/issues). The issue must include the following information:
+The pull request description must include information on the type of change the pull request makes, and what happens in the "call for review" phase. There are three options:
```

Comment:
```suggestion
The pull request description must include information on the type of change the pull request makes and what happens in the "call for review" phase. There are three options:
```
I don't think this is a correct use of a comma (I do this a lot myself and am trying to get out of the habit.). I think the same thing applies to the list below.